airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -211,7 +225,7 @@ def get_field_value(field_value: GoogleAdsRow, field: str, schema_type: Mapping[
             if isinstance(field_value, Enum):
                 field_value = field_value.name
             elif isinstance(field_value, (Repeated, RepeatedComposite)):
-                field_value = [str(value) for value in field_value]
+                field_value = [GoogleAds.serialize_protobuf_message(value) for value in field_value]
```

Comment:
This line may not have the right value you're expecting as input to `serialize_protobuf_message()`. This list comprehension will turn `Repeated`/`RepeatedComposite` into an underlying value within the object. So for this example `Repeated`:

[
  { "text": "An exciting headline", "policy_summary_info": { "approval_status": "APPROVED" } },
  { "text": "Two", "policy_summary_info": { "approval_status": "APPROVED" } }
]

What we end up passing to the method is the first `AdTextAsset`. So in that regard I'm not sure either the method signature, or what we pass your new method is right.

Reply:
Each individual element of the `Repeated` or `RepeatedCoimposite` is indeed passed to the serialization. I'l explore if the whole object serialization will pass the tests.

Reply:
I've just tried to serialize `Repeated` value via the same method using the public 'pb' and got an error that the object does not have "DESCRIPTOR" attribute. If I got it right, it means that the object is not protobuf message compatible with the serializer.

Reply:
ah i see. It feels very weird that we would not end up with a protobuf message at all. What we may want to do instead of relying solely on the try/except block, we should add an explicit type check `isinstance()` and if it not of the correct type, then we defer to `str()`. We can still leave the try catch, but I think proactively checking the type and using str() when its the wrong type is a better pattern

one more question around this. have you been able to test the input for your specific use case? Our tests are semi-stripped down mocks of a real AdRow, before we do final testing and merge, it would be great to verify that given inputs similar to what you have, that we do end up serializing this correctly